### PR TITLE
refactor(footer): update footer layouts

### LINKED ISSUE #401: Improve `footer` component examples
### Provide the exact quote of the error or issue

Several things must be improved in the footer component examples.

### Does the feature already exist? Please link to it.

_No response_

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #401 #315

## Description

I have updated the footer to have better spacing and sizes, and replaced `_container`.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: contents/layouts/footers/footer-links/index.tsx [hunk 1]
```diff
@@ -74,34 +81,17 @@ const FooterLinks: FC = () => {
       bg={["blackAlpha.100", "blackAlpha.500"]}
     >
       <Divider borderColor={["blackAlpha.300", "border"]} />
-      <VStack py="xl" px="md" placeItems="center" maxW="6xl">
+      <VStack gap="0" py="xl" px="md" placeItems="center" maxW="6xl">
         <Flex
```
Comment (illionillion) [inline contents/layouts/footers/footer-links/index.tsx:85]: How about using HStack?

File: contents/layouts/footers/footer-links/index.tsx [hunk 2]
```diff
@@ -46,23 +48,28 @@ const data = [
...
+          <Text fontSize="lg" fontWeight="semibold">
+            {group.title}
+          </Text>
+        </ListItem>
+        <ListItem>
+          <List gap="xs">{links}</List>
+        </ListItem>
+      </List>
     )
   })
```
Comment (illionillion) [inline contents/layouts/footers/footer-links/index.tsx:74]: Wouldn't the code be easier to read if the processing here were separated into a separate component?

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
@illionillion 

Can this PR still be merged?

**Comment by illionillion** [comment]
> @illionillion
> 
> Can this PR still be merged?

OK

**Pull request merged**